src/routes/admin.js

```diff
@@ -0,0 +1,30 @@
+import express from 'express'
+import path from 'path'
+import groupBy from 'lodash/groupBy'
+import * as templateTask from '../database/queries/template_task'
+const router = express.Router()
+
+router.get('/template_task/get', function(req, res, next){
+  templateTask.getAll()
+  .then( results => {
+    const templateTasks = groupBy(results, task => task.user_role)
+    res.json(templateTasks)
+  })
+})
+
+router.post('/template_task/add', function(req, res, next){
```

Comment:
rename this to `/template_tasks`.Since it is a `post` method, it means it is used to create a template task. More info can be found here - http://restfulrouting.com/#golden-seven